src/borrower/feed/nftfeed.sol

```diff
@@ -31,8 +31,9 @@ contract PileLike {
     function changeRate(uint loan, uint newRate) public;
     function loanRates(uint loan) public returns (uint);
     function file(bytes32, uint, uint) public;
-    function rates(uint rate) public view returns(uint, uint, uint ,uint48, uint);
-    function total() public view returns(uint);
+    function rates(uint rate) public view returns (uint, uint, uint ,uint48, uint);
+    function total() public view returns (uint);
```

Comment:
Where are we using this two?

Reply:
for the fv calculation & totalValue in nftFeed